pkg/tools/kubectl_filter.go

```diff
@@ -133,33 +149,39 @@ func analyzeCall(call *syntax.CallExpr) string {
 
 	klog.V(2).Infof("analyzeCall: found kubectl: %q", firstArg)
 
-	// Get the verb (first non-flag argument after kubectl)
-	verbPos := 1 // Start after kubectl at position 0
-	for verbPos < len(args) && strings.HasPrefix(args[verbPos], "-") {
-		verbPos++
+	// Find the verb: first non-flag argument after "kubectl"
+	verb := ""
+	subVerb := ""
+	for i := 1; i < len(args); i++ {
```

Comment:
nit: you can simply use range here because indices are no longer useful in this case :)

```Go

for _, arg := range args {

}

```